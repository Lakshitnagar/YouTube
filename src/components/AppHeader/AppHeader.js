import React from 'react';
import './AppHeader.scss';
import AppLogo from "../AppLogo/AppLogo";
import SearchForm from "../SearchForm/SearchForm";

function AppHeader() {
    return (
        <div className="appHeader">
            <AppLogo/>

            <SearchForm/>

            <div className="appHeader__account">
                accounts
            </div>
        </div>
    );
}

export default AppHeader;
