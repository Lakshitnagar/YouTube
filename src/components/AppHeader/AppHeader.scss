.appHeader {
  display: flex;
  justify-content: space-between;
  align-items: center;
  padding: 0 16px;
  height: 56px;
  box-shadow: 0 1px 5px 1px rgba(0, 0, 0, 0.14);

  .appHeader__account {
  }
}
