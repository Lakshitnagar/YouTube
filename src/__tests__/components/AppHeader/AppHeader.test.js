import React from 'react';
import {shallow} from "enzyme";
import {shallowToJson} from "enzyme-to-json";

import AppHeader from "../../../components/AppHeader/AppHeader";

describe('AppHeader', function () {
    it('should render and match snapshot', function () {
        const component = <AppHeader />;

        const wrapper = shallow(component);

        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });
});
