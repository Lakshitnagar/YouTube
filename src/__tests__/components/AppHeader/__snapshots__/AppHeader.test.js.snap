// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppHeader should render and match snapshot 1`] = `
<div
  className="appHeader"
>
  <AppLogo />
  <Connect(SearchForm) />
  <div
    className="appHeader__account"
  >
    accounts
  </div>
</div>
`;
